HelpSource/Classes/Signal.schelp

```diff
@@ -139,10 +139,7 @@ code::
 Signal.sineFill(512, [1]).plot;
 Signal.sineFill(512, [1]).plot("Signal 1", Rect(50, 50, 150, 450));
 ::
-argument::name
-a String, the name of the window.
-argument::bounds
-a Rect giving the bounds of the window.
```

Comment:
SDDoc only gives warnings when there's a nonzero number of `argument::` tags and there's a count or naming mismatch, so this won't cause a warning. Just tested it to make sure.